tests/utils.py

```diff
@@ -583,8 +590,16 @@ def runAsJava(self, main_code, extra_code=None, args=None):
             )
             out = proc.communicate()[0].decode('utf8')
 
+        if timed:
+            print("  Elapsed time: ", (t1_stop-t1_start) * 1000, " ms")
+            print("  CPU process time: ", (t2_stop-t2_start) * 10000, " ms")
+
         return out
 
+    def runAndBenchAsJava(self, test_name, code):
```

Comment:
I can see what you're doing here, but it's probably better served as 2 lines of code in the external wrapper invoking the code, rather than something on the utility class.

Reply:
Done